app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -186,7 +185,7 @@ class DefaultToolbarMenu(
         appLink(session.content.url).hasExternalApp()
     } ?: false
 
-    private fun shouldShowReaderAppearance(): Boolean = selectedSession?.let {
+    private fun shouldShowReaderViewCustomization(): Boolean = selectedSession?.let {
```

Comment:
nit: this function renaming seems related to the menu refactoring instead of to this PR (i.e. adding 3-dot menu icons). Let's try to keep unrelated changes separate.

Reply:
The reader view PR was already merged, and this rename (from "Appearance" to "Cutomization") was missed. I would like to keep it in this PR so I don't have to open a separate one just for a rename